official/resnet/cifar10_main.py

```diff
@@ -104,20 +104,18 @@ def preprocess_image(image, is_training):
 
 
 def input_fn(is_training, data_dir, batch_size, num_epochs=1,
-             num_parallel_calls=1, multi_gpu=False):
+             use_distribution_strategy=False,
+             gpus_for_distribution_strategy=1):
```

Comment:
(Ditto throughout.)

Reply:
+1 
Think of the params etc that we had as something done in crunch time :) so it would be great to change it to whatever makes most sense for a user.